views/components/ChartList.jsx

```diff
@@ -45,54 +48,78 @@ const ChartList = () => {
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => onDuplicateChart(_id)}
-              >
-                <PlaylistAdd />
-              </Button>
-            </TableCell>
-            <TableCell align='center'>
-              <Link href={routes.editChart(_id)} color='textPrimary'>
-                <Edit />
-              </Link>
-            </TableCell>
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => removeChart(_id)}
-              >
-                <Delete />
-              </Button>
-            </TableCell>
+    <>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align='center'>Title</TableCell>
+            <TableCell align='center'>Description</TableCell>
+            <TableCell align='center'>Duplicate</TableCell>
+            <TableCell align='center'>Edit</TableCell>
+            <TableCell align='center'>Share</TableCell>
+            <TableCell align='center'>Delete</TableCell>
           </TableRow>
-        ))}
-      </TableBody>
-    </Table>
+        </TableHead>
+        <TableBody>
+          {chartList.map((chart) => (
+            <TableRow key={chart._id}>
+              <TableCell align='center'>
+                <Link color='textPrimary' href={routes.chart(chart._id)}>
+                  {chart?.title?.toUpperCase()}
+                </Link>
+              </TableCell>
+              <TableCell align='center'>
+                {chart?.description?.toUpperCase()}
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => onDuplicateChart(chart._id)}
+                >
+                  <PlaylistAdd />
+                </Button>
+              </TableCell>
+              <TableCell align='center'>
+                <Link href={routes.editChart(chart._id)} color='textPrimary'>
+                  <Edit />
+                </Link>
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => handleShareChart(chart)}
+                >
+                  <Share />
+                </Button>
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => removeChart(chart._id)}
+                >
+                  <Delete />
+                </Button>
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+      <ShareChart
+        open={!!chartState._id}
+        owner={user.uid}
+        setDialogState={setChartState}
```

Comment:
this line is confusing to me. Are dialog state and chart state the same? If so, do we need both?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/131/commits/4b7a005d45471fc07236b428f61a2f58b4284d6a